apps/page/option/banned_word.ts

```diff
@@ -75,15 +75,23 @@ export default class BannedWords {
   }
 
   public clear(): void {
-    this.wordList.innerHTML = "";
+    if (this.wordList instanceof HTMLDivElement) {
+      this.wordList.innerHTML = "";
+    } else {
+      Logger.debug("bannedWordList is not HTMLDivElement");
+    }
```

Comment:
If you check in the constructor, the existence check for this element will become unnecessary.